core/actuator.py

```diff
@@ -78,3 +78,10 @@ def _publish(self, message, destination):
         """
         for conn in self.connections:
             conn.publish(message, destination)
+            
+    def _publish_actuator_state(self, message, destination):
```

Comment:
Using this filter for the general 
`def publish(self, message, destination):`
function would break local connections and MQTT connections on the same device (local sensor connected with local actuator via MQTT).

So the only way to join `publish` with `publish_actuator_state` would be to add another parameter to publish. Since I didn't wanted to rewrite other parts of the project which rely on `publish` and I don't know if overloading of the function is possible or desired I made a separate function.

Reply:
Sounds reasonable. I'll keep that in mind next time I decide to to a refactor/rewrite. ;-)